### ISSUE TITLE
Full stranger eye and unique Lost Reels version of the Eye

### ISSUE THREAD

**Issue by Dahoom69** [RiosDeterioratingMentalHealth/OuterWildsPlanetIcons#3]
Hi there, a while ago I made a [post](https://www.reddit.com/r/outerwilds/comments/1ct834o/) about the differences between the multiple eye versions (ironically enough the reason I made the post was to post it here, but I just forgot lol). 
The one from the game seems to be based off the lost reels version somewhat (if the remaining pixels are anything to go by) and there's also that extremely compressed moon(?) symbol that's probably beyond salvation. I hope this will be of some help

I will put the images here for convenience, it is worth noting both stranger eyes have slightly different proportions
lost reels 1200px:
![Lost Reels](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/b0d1adcc-ef52-44cd-b73f-3216076a3c94)
some upscales: 
![2x-UniScale_Restore](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/2b0bfef3-a908-4c8b-a948-478f09e7e3c3)
![1x-Kim2091-DeJpeg-v0-cgi-2x](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/32c21c36-5a6f-4587-bfcb-2e49482e94b3)

**Comment by Dahoom69** [comment]
Follow up to show more images of the stranger moon / full eye, it shows up in a couple of slides and I kinda see it flowing together like the full eye in the repo
![image](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/dc80073b-6bb7-4a90-999d-341f0449ed62)
and upscaled
![image](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/8b02740b-c385-43e1-8041-850efd713585)
![image](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/9d848440-cf17-4ebc-9a69-7b2863256ca1)

**Comment by RiosDeterioratingMentalHealth** [comment]
hii, i'm sowwy, could you pwease restate the issue more pwecisely? i am unsure what exactly you would like to happen.
might be a soft bwain thing on my side @w@

**Comment by Dahoom69** [comment]
Oh wow, not sure how I didn't notice your reply, my bad, could've sworn I checked the replies every so often.

A while ago, I noticed that the eye of the universe logo used in the game is actually incomplete, and I guess "censored" 2 devs even liked my Twitter post when I pointed it out cuz apparently no one noticed before. This is just me trying to point it out here because I think the stranger eye is worth getting the same treatment as the nomai one, extracted textures from the game, tried upscaling them just to make things more readable slightly. The one from the lost reels album is also different and has more detail visible in the center, and I feel it's worth adding as its own svg in the repo. I really love the stranger's eye symbol and want to use it in wallpapers and such.

 TLDR; I would really appreciate it if these unique versions of the eye (lost reels album, the seal, and the full eye) were added eventually.
 
Also, if you want, I can datamine the game for any other icon textures you want for accuracy and stuff, even character clothing and such. If you want to, then you can contact me on discord "rayanwasalsotaken"

**Comment by Dahoom69** [comment]
Here are the images from the above mentioned post just for better reference

![Schizo Ramblings Page 1](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/e0180ce7-193e-4cff-8b9b-201fef2a539d)
![Schizo Ramblings Page 2](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/0762568f-faf3-461f-a5c2-78bda4b92293)
![Schizo Ramblings Page 3](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/assets/72060150/9fa162f2-911b-4be6-b69c-ad57e1b18fb0)

**Comment by Dahoom69** [comment]
@RiosDeterioratingMentalHealth basically reviving a dead thread, but the new vinyl release has a page dedicated to the new dlc eye design. Thanks to [Kristen](https://x.com/Kristen_Ropars/status/1845843567266124037) for the photo. Thought you might find it interesting.
![GZ3A4Q0W4BENWch](https://github.com/user-attachments/assets/a163584b-06ef-4cae-840a-fd8e6ef3324d)

**Comment by RiosDeterioratingMentalHealth** [comment]
@Dahoom69 sorry for the super late reply :<
i dont have many capacities, so i cant promise to look into it, altho this does sound interesting. 

if you or someone else creates an svg icon of the full stranger eye, feel free to make a merge request. i'll try to approve it without too much delay.

**Comment by Dahoom69** [comment]
@RiosDeterioratingMentalHealth no worries, that actually works in our favor, a friend of mine bought the secrets of the outer wilds vinyl where this symbol comes from, so I'm just waiting for him to get a scanner because it will be much more accurate than a phone camera. 
I have been meaning to give it a shot myself as well, but I've never done this sort of stuff before so it might take even longer, and I'm not sure if it will even turn out good lol.

**Comment by randomstatistic** [comment]
Add my vote for the album's Trapped Eye. I've been playing with making Outer Wilds content on a 3d printer this week, and although I can get by with pictures or screenshots for some things, I was delighted to find this SVG archive. The difference in the detail gets apparent quickly.
Here's a lantern I used a jpg of the album cover for:

![Image](https://github.com/user-attachments/assets/6c8a6bbc-fad9-466b-a99c-803e18ccaef9)

By contrast, here's a coaster using your SVG.

![Image](https://github.com/user-attachments/assets/b8a860d6-d815-4ada-b530-908f5ef85bc2)

**Comment by RiosDeterioratingMentalHealth** [comment]
hey @Dahoom69, i got my shit together and created a full version of the stranger eye ^.^

maybe i will also add the trapped eye in a few days or weeks, but no promises ;3

[stranger-eye/full-stranger-eye.png](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/blob/cd8b8c133ae8a128082be91c7b7b8f784ac08213/stranger-eye/full-stranger-eye.png)

it's not as good as the other ones maybe, but i did not have as much to go off either

**Comment by RiosDeterioratingMentalHealth** [comment]
heyy @Dahoom69 and @randomstatistic, i added the trapped eye now. 

[stranger-eye/trapped-stranger-eye.png](https://github.com/RiosDeterioratingMentalHealth/OuterWildsPlanetIcons/blob/60efc08f7927d729b15587ac28adfa95bfa72798/stranger-eye/trapped-stranger-eye.png)

with that i would close this issue ::3
